### PR TITLE
Quantile Huber Loss to predict Specific Quantile Value

### PR DESCRIPTION
**Pull request by ashutosh1919**
We know that we have huber loss function added in keras tf-2 already which can perform both kind of behaviour MSE and MAE depending on the scale of data. In most of the prediction and analysis models, we often do not need just median or mean predicted value, but we also need the specific quantile value of prediction. This is the loss function I am adding which is derived from **huber_loss** here 
 [https://arxiv.org/pdf/1402.4624.pdf](here). I name it **quantile_huber_loss** and the class which calls it is QuantileHuber. I have given great attention to the detail in implementing the function. function takes two extra arguements **delta and quantile**, where delta is common as the huber_loss and quantile is the float number between 0 and 1. Equation looks as below.
qhuber_form

### CODE REVIEW

**Comment by ashutosh1919** [comment]
@abhinavsagar @fchollet Please Review this PR. I have added QuantileHuber loss. As per my observation, this is first Quantile loss function added in keras. This branch has no conflict with the tf-2 branch. Although, Some default integration failures are there. Please review this PR.

**Comment by abhinavsagar** [comment]
Could you fix the build errors.

**Comment by ashutosh1919** [comment]
There are default build errors even before I added my code. Although, I have fixed the issues related to BinaryCrossEntropy and its tests. There are no additional failures other than default ones. @abhinavsagar

**Comment by abhinavsagar** [comment]
LGTM

Cc @fchollet

**Comment by fchollet** [comment]
Thank you for preparing this PR.

We are no longer adding new features to multi-backend Keras, as we are refocusing development efforts on `tf.keras`. If you are still interested in submitting this PR, please direct it to `tf.keras` in the TensorFlow repository instead.

**Comment by ashutosh1919** [comment]
> Thank you for preparing this PR.
> 
> We are no longer adding new features to multi-backend Keras, as we are refocusing development efforts on `tf.keras`. If you are still interested in submitting this PR, please direct it to `tf.keras` in the TensorFlow repository instead.

Thank you @fchollet , I will directly add it into tf.keras.
